backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -627,4 +594,10 @@ private String generateRoleSuffix(OrganizationRole role) {
   private String generateFacilitySuffix(String facilityId) {
     return ":" + OrganizationExtractor.FACILITY_ACCESS_MARKER + ":" + facilityId;
   }
+
+  private void isEmpty(Stream<?> stream, String errorMessage) {
+    if (stream.findAny().isEmpty()) {
```

Comment:
😊

Reply:
Super nitpicky question here - `isEmpty` seems like a misleading function name. I expect something like `isEmpty` to return a boolean value, but it is kind of a side effect function (throwing exception) in addition to checking if empty. Would it be possible to name this to clarify the behavior? like `throwErrorIfEmpty` or something?

Reply:
Not nitpicky at all I'd say. I was thinking the name wasn't that great, thanks for the suggestion!

Reply:
😄